src/main/java/com/aws/greengrass/mqttbridge/util/BatchedSubscriber.java

```diff
@@ -48,11 +48,11 @@
             what == WhatHappened.timestampUpdated || what == WhatHappened.interiorAdded;
 
     private final AtomicInteger numRequestedChanges = new AtomicInteger();
-    private final Set<Node> whatChanged = new HashSet<>();
+    private final List<Node> whatChanged = new ArrayList<>();
```

Comment:
If you want to maintain order, use a LinkedHashSet

Reply:
👍  per other comment, going to switch back to set.